### DISCUSSION TITLE
Install Guage / Java locally without admin rights

### DISCUSSION THREAD

**Discussion by tpeterson1-jeff** [getgauge/gauge#2434] [General]
Hi,
Is there any way to install Gauge for Java on Windows without the installer? I do not have local admin rights on my machine and getting an exception is a nearly impossible feat in the organization. I need to just be able to run it locally from my user directory.  Also, I do not use (and am unable to install) VS Code. I have IntelliJ installed and I see there is an IntelliJ plugin for Gauge.

Thanks,
Troy

**Comment by zabil** [comment]
There's a zip version for windows with the `gauge` binary at the following location.
https://github.com/getgauge/gauge/releases
